### PR TITLE
Add benchmark for hybrid composition on Android

### PR DESCRIPTION
**Pull request by blasten**
This change adds a new benchmark for hybrid composition on Android.  Therefore, there will be benchmarks for the Android texture and the Android view.

### CODE REVIEW

File: dev/benchmarks/platform_views_layout/lib/android_platform_view.dart [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  }
+
+  PlatformViewController _onCreateAndroidView(PlatformViewCreationParams params) {
+    final _AndroidViewController controller = _AndroidViewController(params.id, viewType);
+    controller._initialize().then((_) { params.onPlatformViewCreated(params.id); });
+    return controller;
+  }
+}
+
+// TODO(egarciad): The Android view controller should be defined in the framework.
```
Comment (cyanglaz) [inline dev/benchmarks/platform_views_layout/lib/android_platform_view.dart:48]: What is the motivation of this change? (I mean add AndroidViewController to the framework)
Also, please link an issue for the TODOs, this one and the below ones.

**Review by cyanglaz — CHANGES_REQUESTED** [review]

File: dev/benchmarks/platform_views_layout/lib/android_platform_view.dart [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+
+class AndroidPlatformView extends StatelessWidget {
```
[the remark this answers is not in the document]
Comment (blasten) [reply dev/benchmarks/platform_views_layout/lib/android_platform_view.dart:11]: The Dart doc is in the constructor. The doc can go in either place.

File: dev/benchmarks/platform_views_layout/lib/android_platform_view.dart (hunk 1 above)
Comment (blasten) [reply dev/benchmarks/platform_views_layout/lib/android_platform_view.dart:48]: I added a link to https://github.com/flutter/flutter/issues/55904. The other TODOs don't need an individual issue since they are all related to the same issue.

**Review by cyanglaz — APPROVED** [review]
LGTM!
